### PR TITLE
Port System.Security.Cryptography triple slash to docs

### PR DESCRIPTION
**Pull request by carlossanlop**
Ported with DocsPortingTool.

@gewarren targeting the net6-rc1 branch.

### CODE REVIEW

**Comment by gewarren** [comment]
Docs Build status updates of commit _[5689817](https://github.com/carlossanlop/dotnet-api-docs/commits/56898178cbc176842475787f364fdf959ce57e93)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/System.Security.Cryptography/ChaCha20Poly1305.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/ChaCha20Poly1305.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.chacha20poly1305?view=net-6.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/CngProvider.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/CngProvider.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.cngprovider?view=dotnet-plat-ext-2.1&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/ECDiffieHellmanPublicKey.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/ECDiffieHellmanPublicKey.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ecdiffiehellmanpublickey?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACMD5.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACMD5.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacmd5?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACSHA1.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACSHA1.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacsha1?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACSHA256.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACSHA256.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacsha256?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACSHA384.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACSHA384.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacsha384?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACSHA512.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACSHA512.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacsha512?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/RandomNumberGenerator.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/RandomNumberGenerator.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.randomnumbergenerator?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rfc2898derivebytes?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/SymmetricAlgorithm.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/SymmetricAlgorithm.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.symmetricalgorithm?view=net-5.0&branch=pr-en-us-7098) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202109030657023210-7098%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=uPB0QDN9B2JehAZTZPJ39%2F%2Fu%2F%2BU2rsaaOF%2Bah4L43B4%3D&st=2021-09-03T07%3A25%3A46Z&se=2021-10-04T07%3A30%3A46Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by gewarren** [comment]
Docs Build status updates of commit _[2fd68a4](https://github.com/carlossanlop/dotnet-api-docs/commits/2fd68a4fb1003f1eb93116f36ad8717229da58f0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/System.Security.Cryptography/ChaCha20Poly1305.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/ChaCha20Poly1305.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.chacha20poly1305?view=net-6.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/CngProvider.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/CngProvider.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.cngprovider?view=dotnet-plat-ext-2.1&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/ECDiffieHellmanPublicKey.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/ECDiffieHellmanPublicKey.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ecdiffiehellmanpublickey?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACMD5.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACMD5.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacmd5?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACSHA1.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACSHA1.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacsha1?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACSHA256.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACSHA256.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacsha256?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACSHA384.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACSHA384.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacsha384?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/HMACSHA512.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/HMACSHA512.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hmacsha512?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/RandomNumberGenerator.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/RandomNumberGenerator.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.randomnumbergenerator?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rfc2898derivebytes?view=net-5.0&branch=pr-en-us-7098) |
[xml/System.Security.Cryptography/SymmetricAlgorithm.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.sec.cryp/xml/System.Security.Cryptography/SymmetricAlgorithm.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.symmetricalgorithm?view=net-5.0&branch=pr-en-us-7098) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202109030749138175-7098%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=po2T4Pdj2ByeOBvfGWh%2BqzghjQMj9LGvd4Sn2%2B%2BLo1Y%3D&st=2021-09-03T08%3A14%3A42Z&se=2021-10-04T08%3A19%3A42Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: xml/System.Security.Cryptography/HMACSHA512.xml [hunk 1]
```diff
@@ -625,12 +628,13 @@ public static void Test()
...
-        <param name="key">To be added.</param>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">The HMAC key.</param>
+        <param name="source">The data to HMAC.</param>
+        <param name="destination">The buffer to receive the HMAC value.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written into <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/HMACSHA512.xml:634]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written into <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml [hunk 2]
```diff
@@ -873,14 +873,23 @@
...
+        <param name="hashAlgorithm">The hash algorithm to use to derive the key.</param>
+        <param name="outputLength">The size of key to derive.</param>
+        <summary>Creates a PBKDF2 derived key from password bytes.</summary>
         <returns>To be added.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="password" /> or <paramref name="salt" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="outputLength" /> is not zero or a positive value.
+-or- <paramref name="iterations" /> is not a positive value.</exception>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml:888]:
```suggestion

-or-

<paramref name="iterations" /> is not a positive value.</exception>
```

File: xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml [hunk 3]
```diff
@@ -912,14 +921,21 @@
...
+        <param name="salt">The key salt used to derive the key.</param>
+        <param name="iterations">The number of iterations for the operation.</param>
+        <param name="hashAlgorithm">The hash algorithm to use to derive the key.</param>
+        <param name="outputLength">The size of key to derive.</param>
+        <summary>Creates a PBKDF2 derived key from password bytes.</summary>
         <returns>To be added.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="outputLength" /> is not zero or a positive value.
+-or- <paramref name="iterations" /> is not a positive value.</exception>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml:934]:
```suggestion

-or-

<paramref name="iterations" /> is not a positive value.</exception>
```

File: xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml [hunk 4]
```diff
@@ -989,14 +1011,33 @@
...
+
+The `password` will be converted to bytes using the UTF8 encoding. For
+            other encodings, convert the password string to bytes using the appropriate <xref:System.Text.Encoding>
+            and use <xref:System.Security.Cryptography.Rfc2898DeriveBytes.Pbkdf2(System.ReadOnlySpan{System.Byte},System.ReadOnlySpan{System.Byte},System.Int32,System.Security.Cryptography.HashAlgorithmName,System.Int32)>.
+
+          ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="outputLength" /> is not zero or a positive value.
+-or- <paramref name="iterations" /> is not a positive value.</exception>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml:1034]:
```suggestion

-or-

<paramref name="iterations" /> is not a positive value.</exception>
```

File: xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml [hunk 5]
```diff
@@ -1066,14 +1125,35 @@
...
+            other encodings, convert the password string to bytes using the appropriate <xref:System.Text.Encoding>
+            and use <xref:System.Security.Cryptography.Rfc2898DeriveBytes.Pbkdf2(System.Byte[],System.Byte[],System.Int32,System.Security.Cryptography.HashAlgorithmName,System.Int32)>.
+
+          ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="password" /> or <paramref name="salt" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="outputLength" /> is not zero or a positive value.
+-or- <paramref name="iterations" /> is not a positive value.</exception>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/Rfc2898DeriveBytes.xml:1150]:
```suggestion

-or-

<paramref name="iterations" /> is not a positive value.</exception>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 6]
```diff
@@ -2487,14 +2853,24 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="paddingMode">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="ciphertext">The data to decrypt.</param>
+        <param name="iv">The initialization vector.</param>
+        <param name="destination">The buffer to receive the plaintext data.</param>
+        <param name="paddingMode">The padding mode used to produce the ciphertext and remove during decryption.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:2860]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 7]
```diff
@@ -2526,15 +2902,37 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="bytesWritten">To be added.</param>
-        <param name="paddingMode">To be added.</param>
-        <param name="feedbackSizeInBits">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="ciphertext">The data to decrypt.</param>
+        <param name="iv">The initialization vector.</param>
+        <param name="destination">The buffer to receive the plaintext data.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:2908]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 8]
```diff
@@ -2567,15 +2965,25 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="ciphertext">The data to decrypt.</param>
+        <param name="iv">The initialization vector.</param>
+        <param name="destination">The buffer to receive the plaintext data.</param>
+        <param name="paddingMode">The padding mode used to produce the ciphertext and remove during decryption.</param>
+        <param name="feedbackSizeInBits">The feedback size, specified in bits.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:2973]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 9]
```diff
@@ -2606,13 +3014,25 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="destination">To be added.</param>
-        <param name="paddingMode">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="ciphertext">The data to decrypt.</param>
+        <param name="destination">The buffer to receive the plaintext data.</param>
+        <param name="paddingMode">The padding mode used to produce the ciphertext and remove during decryption.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3020]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 10]
```diff
@@ -2643,13 +3063,23 @@ This member is an explicit interface member implementation. It can be used only
...
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3069]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 11]
```diff
@@ -2680,14 +3110,28 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <param name="paddingMode">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="plaintext">The data to encrypt.</param>
+        <param name="iv">The initialization vector.</param>
+        <param name="destination">The buffer to receive the ciphertext data.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3116]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 12]
```diff
@@ -2719,14 +3163,32 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="paddingMode">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="plaintext">The data to encrypt.</param>
+        <param name="iv">The initialization vector.</param>
+        <param name="destination">The buffer to receive the ciphertext data.</param>
+        <param name="paddingMode">The padding mode used to produce the ciphertext and remove during decryption.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3170]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 13]
```diff
@@ -2719,14 +3163,32 @@ This member is an explicit interface member implementation. It can be used only
...
+          <see langword="true" /> if <paramref name="destination" /> was large enough to receive the encrypted data; otherwise, <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Derived classes must override this and provide an implementation.
+
+
+                 Implementations of this method must write precisely
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3182]:
```suggestion
Implementations of this method must write precisely
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 14]
```diff
@@ -2758,15 +3220,37 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="bytesWritten">To be added.</param>
-        <param name="paddingMode">To be added.</param>
-        <param name="feedbackSizeInBits">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="plaintext">The data to encrypt.</param>
+        <param name="iv">The initialization vector.</param>
+        <param name="destination">The buffer to receive the ciphertext data.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3226]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 15]
```diff
@@ -2799,15 +3283,33 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="plaintext">The data to encrypt.</param>
+        <param name="iv">The initialization vector.</param>
+        <param name="destination">The buffer to receive the ciphertext data.</param>
+        <param name="paddingMode">The padding mode used to produce the ciphertext and remove during decryption.</param>
+        <param name="feedbackSizeInBits">The feedback size, specified in bits.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3291]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 16]
```diff
@@ -2799,15 +3283,33 @@ This member is an explicit interface member implementation. It can be used only
...
+          <see langword="true" /> if <paramref name="destination" /> was large enough to receive the encrypted data; otherwise, <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Derived classes must override this and provide an implementation.
+
+
+                 Implementations of this method must write precisely
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3303]:
```suggestion
Implementations of this method must write precisely
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 17]
```diff
@@ -2838,13 +3340,25 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="destination">To be added.</param>
-        <param name="paddingMode">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="plaintext">The data to encrypt.</param>
+        <param name="destination">The buffer to receive the ciphertext data.</param>
+        <param name="paddingMode">The padding mode used to produce the ciphertext and remove during decryption.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3346]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 18]
```diff
@@ -2875,13 +3389,31 @@ This member is an explicit interface member implementation. It can be used only
...
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3395]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 19]
```diff
@@ -2875,13 +3389,31 @@ This member is an explicit interface member implementation. It can be used only
...
+          <see langword="true" /> if <paramref name="destination" /> was large enough to receive the encrypted data; otherwise, <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Derived classes must override this and provide an implementation.
+
+
+                 Implementations of this method must write precisely
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:3407]:
```suggestion
Implementations of this method must write precisely
```

File: xml/System.Security.Cryptography/SymmetricAlgorithm.xml [hunk 20]
```diff
@@ -2448,14 +2800,28 @@ This member is an explicit interface member implementation. It can be used only
...
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <param name="paddingMode">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="ciphertext">The data to decrypt.</param>
+        <param name="iv">The initialization vector.</param>
+        <param name="destination">The buffer to receive the plaintext data.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```
Comment (gewarren) [inline xml/System.Security.Cryptography/SymmetricAlgorithm.xml:2806]:
```suggestion
        <param name="bytesWritten">When this method returns, contains the total number of bytes written to <paramref name="destination" />.</param>
```

**Review by gewarren — APPROVED** [review]

**Comment by carlossanlop** [comment]
The error is not telling me which file is missing a `name` attribute. I double checked the last commit, none of the changes looked wrong.

**Comment by gewarren** [comment]
I think that's a bogus error. I'll close/reopen.

**Pull request merged**